### PR TITLE
fix mobile font size issue

### PR DESCRIPTION
**Pull request by RamRamez**
Related to PR [#68](https://github.com/Giveth/ui-design-system/pull/68)

1. fix button color on about page
2. make accordion header unselectable
3. fix faq padding and title z-index on mobile
4. center align support card
5. make Submission guidelines card unselectable
6. convert script tags in blog posts title to text
7. fix Changemakers title and text alignment in mobile
8. fix FROM OUR BLOG title font type
9. fix Join our Community font size on mobile
10. update @giveth/ui-design-system
11. fix Partnerships font size on mobile
12. fix project card updated at position on mobile
13. fix projects title going under arc
14. fix waiting for review not showing correctly on mobile
15. fix Terms of use wrapper responsive margin and padding

### CODE REVIEW

File: src/components/views/about/AboutIndex.tsx [hunk 1]
```diff
@@ -223,9 +226,8 @@ const Middle = styled.div`
 	}
 `;
 
-const UpperButton = styled(ButtonLink)`
-	margin: 50px auto 180px auto;
-	width: fit-content;
+const UpperButton = styled(Button)`
+	margin: 50px auto 180px;
```
Comment (MohammadPCh) [inline src/components/views/about/AboutIndex.tsx:230]: we should use ButtonLink.

**Review by MohammadPCh — CHANGES_REQUESTED** [review]

File: src/components/views/about/AboutIndex.tsx (hunk 1 above)
Comment (RamRamez) [reply src/components/views/about/AboutIndex.tsx:230]: It's a button, what's the difference? why we should use ButtonLink?

Comment (MohammadPCh) [reply src/components/views/about/AboutIndex.tsx:230]: It's a link to another page not a button.

**Review by MohammadPCh — APPROVED** [review]
LGTM @RamRamez

**Pull request merged**